azure-pipelines-docs-integration.yml

```diff
@@ -123,7 +123,7 @@ stages:
 
         - script: |
             pip install pytest pytest-azurepipelines
-            pytest -v --docs-tests -m docs --mysql --mssql tests/integration/test_script_runner.py
+            pytest -v --docs-tests -m docs --mysql --bigquery --mssql tests/integration/test_script_runner.py
```

Comment:
docs tests now run with the `--bigquery` flag

Reply:
Is this getting us one step closer to being able to run --docs-tests locally>

Reply:
Yes, well the functionality was there before too, but was a bit finicky.  For GCP-related tests they will all respond to the `--bigquery` flag so they can be turned off and on